src/app/(main)/query-runner/_components/cql-editor.tsx

```diff
@@ -40,6 +46,8 @@ enum ExecuteType {
   CURRENT = 1,
 }
 
+const isMacEnviroment = getIsMacEnviroment(window.navigator.userAgent);
```

Comment:
As shown in the pipeline, the global variable `window` isn't defined on this context.

![image](https://github.com/user-attachments/assets/cab97ea0-158b-4892-bf47-abb7b193c323)

I saw on [next-docs](https://nextjs.org/docs/14/app/api-reference/functions/userAgent#os) they have a `userAgent` function, but idk if it will work in our context.